src/components/customSearch/AddFieldOrGroupButton.tsx

```diff
@@ -0,0 +1,31 @@
+import { Button } from '@mui/material'
+import { SearchParameter } from '../../types/SimpleSearch'
+import AddIcon from '@mui/icons-material/Add'
+
+interface AddFieldOrGroupButtonProps {
+  push: (obj: any) => void
+  initialCustomSearchValues: SearchParameter
+  label: string
+}
+const AddFieldOrGroupButton: React.FC<any> = ({
+  push,
+  initialCustomSearchValues,
+  label
+}) => {
+  return (
+    <>
```

Comment:
We don't need fragment unless there's multiple elements.

Reply:
sure